cortexapps_cli/cortex.py

```diff
@@ -106,6 +106,9 @@ def check_config_file(config_file, replace_string):
 # config file.
 def get_config(config, args, argv, parser, replace_string):
     if os.environ.get('CORTEX_API_KEY'):
+        if args.tenant:
+            print("WARNING: tenant setting overidden by CORTEX_API_KEY")
+
```

Comment:
Per our slack chat, this is breaking some tests because of the way STDOUT is parsed.

And good idea to add that message.  Maybe write to STDERR?

Reply:
oh i just need to send this to stderr, yes?

Reply:
yeah this was a point of confusion for me, i'll do that.